tests/staging/test_data_cleaning.py

```diff
@@ -104,13 +104,16 @@ def test_run_live_or_frozen(filepath):
 
     df = pd.read_csv(filepath / "test_run_live_or_frozen.csv")
 
-    df_in = df.drop(columns=["frozen"])
+    df_in = df.drop(columns=["frozen", "frozen_error"])
```

Comment:
Guessing you are using one input data to have the input and correct output which is why its being dropper early in test?